cluster/examples/kubernetes/ceph/create-external-cluster-resources.py

```diff
@@ -219,7 +219,9 @@ def get_ceph_external_mon_data(self):
 
     def get_active_ceph_mgr(self):
         if self._arg_parser.monitoring_endpoint:
-            return self._arg_parser.monitoring_endpoint+':'+self._arg_parser.monitoring_endpoint_port
+            monitoring_endpoint = ':'.join([
```

Comment:
Is there a more IP and Port agnostic Python function available to join a IP / host and port?

E.g., Golang has the `net.JoinHostPort` function https://golang.org/pkg/net/#JoinHostPort

Reply:
Same comment.

Reply:
@galexrt , @leseb , I was unable to find a `join()` method equivalent to golang's `net.JoinHostPort()`. I created one with the spec from https://golang.org/pkg/net/#JoinHostPort .
If you could find one similar (without introducing a third party lib) let me know...

Reply:
Maybe this one https://docs.python.org/2/library/urlparse.html?

Reply:
Used the `urlparse` lib (even made it compatible with python 2.x and 3.x) to parse the endpoints, but it doesn't have a join method. There is a [urljoin](https://docs.python.org/2/library/urlparse.html#urlparse.urljoin), but it serves a different purpose.